drivers/wifi/nrf700x/zephyr/src/zephyr_twt.c

```diff
@@ -74,6 +74,14 @@ int wifi_nrf_set_twt(const struct device *dev,
 	case WIFI_TWT_TEARDOWN:
 		twt_info.twt_flow_id = twt_params->flow_id;
 
+		if (twt_params->teardown.teardown_all) {
+			/* update the negotiated flow id from the negotaited value. */
+			twt_info.twt_flow_id = vif_ctx_zep->negotiated_twt_flow_id;
+		} else {
+			/* update the flow id as given by the user. */
```

Comment:
Use capital letter for the first letter in a sentence (i.e. update should be Update)